sdk/loadtestservice/load-testing-rest/swagger/README.md

```diff
@@ -13,45 +13,61 @@ generate-test: true
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../
 source-code-folder-path: ./src
-input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/loadtestservice/data-plane/Microsoft.LoadTestService/preview/2022-06-01-preview/loadtestservice.json
-package-version: 1.0.0-beta.1
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/loadtest_stableapis/specification/loadtestservice/data-plane/Microsoft.LoadTestService/stable/2022-11-01/loadtestservice.json
+package-version: 1.0.0
 rest-level-client: true
 security: AADToken
 security-scopes: "https://cnt-prod.loadtesting.azure.com/.default"
 use-extension:
   "@autorest/typescript": "6.0.0-rc.1"
```

Comment:
Can you use latest dev release of @autorest/typescript and regenerate the code 6.0.0-rc.3.20221116.2

Reply:
regenerated code using @autorest/typescript - 6.0.0-rc.3